cmd/weaver-gke/store.go

```diff
@@ -27,23 +27,23 @@ import (
 
 var (
 	storeFlags  = newCloudFlagSet("store", flag.ContinueOnError)
-	storeRegion = storeFlags.String("region", gke.ConfigClusterRegion,
+	storeRegion = storeFlags.String("region", "",
 		`Cloud region where the store resides. Default value is the region of
```

Comment:
Delete the sentence starting with `Default value...`

Reply:
Done